src/lib/google-auth/GoogleAuth.svelte

```diff
@@ -1,4 +1,8 @@
-<button bind:this={signinCta} {disabled} use:action>
+<button
+  {disabled}
+  use:action
+  on:click={handleClick}
```

Comment:
We should call event handler functions after what they do, not what events they handle

Reply:
The click here triggers the Google Sign In popup. Would the function name `triggerPopup` work for you?

Reply:
Renamed to `triggerPrompt` ✅